autoextract_poet/exceptions.py

```diff
@@ -0,0 +1,9 @@
+from dataclasses import dataclass
+from typing import List
+
+
+@dataclass
+class QueryLevelError(Exception):
```

Comment:
This exception has way more attributes on its original form. `request` attribute was removed because it was not being used. `url` and `pageType` attributes were removed because they're already contained within the `query` attribute.

Reply:
> request attribute was removed because it was not being used

I think you mean "response". I see I'm not saving it in the QueryLevelError, but I think it could be useful to keep the full response there. 

> url and pageType attributes were removed because they're already contained within the query attribute.

Understand your point here. I'm unsure about what is best to do with a case like that. 
`query` contains all the parameters as you pointed but at the same time is not making explicit what is inside so it is more difficult for developers to know what it contains (they need to read the AutoExtract service documentation). That's what I opted to make attributes like `url` and `pageType` explicit.

Reply:
Yes, it's supposed to be `response` and not `request` 😅 
Yes, the values appear in a list here, that's how the query is built 😕  
But think it should easy to spot what's the URL and what's the AutoExtract page type.
Does that make sense?

Reply:
After some refactoring, exception should look like this on console:

```python
autoextract_poet.utils.QueryLevelError: (Query(url='example.com', page_type='myPageType', full_html=True), 'error')
```